src/file-size/format.js

```diff
@@ -0,0 +1,20 @@
+import FileSizeFormatter from './format-file-size';
+import localeProvider from '../locale-provider';
+
+export default class FileSizeFormat {
+	constructor(locale, options) {
+		this.options = options || {};
+		this.localeData = localeProvider(locale, options.locale);
+	}
+	formatFileSize(fileSize, options){
```

Comment:
Can this method just be called `format`? That way it's consistent with the number and date formatters.